lips/lip-8.md

```diff
@@ -125,12 +139,73 @@ The structure in this object can be a full pre-approval or just the fields of an
 }
 ```
 
+### ScopeObject
+
+In this object the initiator VASP declares its intent for the pre-approval, this can be one of two options:
+  1. Save the consumer sub-account for future transactions (save_sub_account)- this will enable the initiator VASP (merchant) to charge the sub-account in the future but will require the owner to approve the transaction. When using this option, amount limits are not required
+  2. Save the consumer sub-account and get a consent for future payments (consent) - this enables the initiator VASP (merchant) to charge the sub-account without any interaction with the owner. 
+Also, the scope limits the FundPullPreApprovalObject to certain parameters of time and amount. This object can be later mutated by the initiator VASP if needed, but any change requires the target VASP to approve the scope change.
```

Comment:
what is "sub_account" means?
use name "type" is confusing here, as type indicates the object type, which is protocol level,
but it looks like a business agreement about how the next payment object can be processed.

Reply:
"sub_account" is the consumer identifier within (behind) the VASP. 
It might be a good idea to replace it with another term that indicates the wallet user.

Reply:
ahh, interesting insight, we should determine a uniform fashion to call these out. I am concerned that subaccount doesn't really refer to a client's account and I've heard a lot of confusion across subaddress and child vasps :/